docs/_docs/troubleshooting.md

```diff
@@ -40,3 +40,21 @@ Running database migrations
 ```
 
 This warning means that some migrations that were previously executed do not exist within the package. This may be because they were deleted or potentially renamed. If these are core migrations (`jadu/cms` or `jadu/xfp`) then check with Support as this should not happen. To avoid this happening do not rename migration verisons after a patch has been applied. If you need to remove a migration then simply empty the `up` and `down` methods.
+
+### Database connection error
+
+If the first stages of deployment take a long time (e.g. due to network latency) then it is possible that the database
+connection may timeout before the migrations are applied. In this case you may see and error such as:
+
+```
+Error: PDOException
+SQLSTATE[HY000]: General error: 2006 MySQL server has gone away
```

Comment:
Could this happen for other kinds of servers? e.g. if we patch WISP, wouldn't the message include something like `SQL Server has gone away`?

Reply:
Less likely - as you mentioned this predominantly happens with LAMP environments using NFS